README.md

```diff
@@ -0,0 +1,32 @@
+# Sitecore Headless Samples and Examples
+This repository provides Sitecore and community-developed examples of various implementation and integration scenarios, for implementations which utilize Sitecore Headless SDKs. While Sitecore already provides official boilerplate and starters for these frameworks, questions about how to do X or Y often come up in community channels. The aim for this repository is to build a library of samples to help address these questions and provide a reference for Sitecore developers.
+
+## Where can I learn more about Sitecore Headless SDKs?
+* [Sitecore JavaScript Rendering SDK](https://jss.sitecore.com/)
+* [Sitecore ASP.NET Core Rendering SDK](https://doc.sitecore.com/developers/101/developer-tools/en/sitecore-headless-development.html)
```

Comment:
This links to the docs on Headless Services - maybe that should be called out on it's own?
.NET Core SDK page = https://doc.sitecore.com/developers/101/developer-tools/en/sitecore-asp-net-rendering-sdk.html

Reply:
Change to: 
```
* [Sitecore JavaScript Rendering SDK](https://doc.sitecore.com/en/developers/hd/190/sitecore-headless-development/sitecore-javascript-rendering-sdks--jss-.html)
* [Sitecore ASP.NET Core Rendering SDK](https://doc.sitecore.com/en/developers/hd/190/sitecore-headless-development/sitecore-asp-net-rendering-sdk.html)
```